census_example.ipynb

```diff
@@ -179,8 +194,8 @@
     "clf.fit(\n",
     "    X_train=X_train, y_train=y_train,\n",
     "    X_valid=X_valid, y_valid=y_valid,\n",
-    "    max_epochs=1000, patience=50,\n",
-    "    batch_size=1024, virtual_batch_size=128\n",
+    "    max_epochs=1000, patience=10,\n",
```

Comment:
Why patience 10? I think convergence is sometimes slow and somewhat long ups and downs on the validation metric have been observed. Is the goal is to have a notebook that runs quicker than the previous one?

Reply:
@eduardocarvp Yep, goal was to have a better performance with a shorter time.

Feel free to test and propose changes if you feel like this is bad

Reply:
We made some changes about which columns to consider as categorical and it improved quite significantly the results so I guess we're good now, I'm closing this!